Code/Framework/AzTest/AzTest/AzTest.h

```diff
@@ -448,32 +448,18 @@ int main(int argc, char** argv)
 
 #define DEFAULT_UNIT_TEST_ENV AZ::Test::DefaultTestEnv()
 
-#define AZ_UNIT_TEST_HOOK_1(_1)                                                                                  \
-    AZ_UNIT_TEST_HOOK_ENV(_1)                                                                                    \
-    AZ_BENCHMARK_HOOK()                                                                                          \
+#define AZ_UNIT_TEST_HOOK_1(_1)                                                                                                            \
```

Comment:
Can you move these changes to AzTest to another branch?

These will be useful in the future

Reply:
I don't creating a branch as reference, sure

Reply:
The branch is [aztest_hook_dynamic](https://github.com/aws-lumberyard-dev/o3de/tree/aztest_hook_dynamic).

Reply:
Thanks!